client/ayon_core/hosts/houdini/startup/MainMenuCommon.xml

```diff
@@ -104,6 +104,34 @@ parent = hou.qt.mainWindow()
 host_tools.show_experimental_tools_dialog(parent)
 ]]></scriptCode>
             </scriptItem>
+
+        <separatorItem/>
+
+        <scriptItem id="ayon_server">
+            <label>Open AYON server in browser</label>
+            <scriptCode><![CDATA[import os; __import__("webbrowser").open(os.getenv("AYON_SERVER_URL"))]]></scriptCode>
+        </scriptItem>
+
+        <subMenu id="ayon_help">
+            <label>Help...</label>
```

Comment:
I've never seen submenus with trailing dots?
```suggestion
            <label>Help</label>
```

The trailing dots usually indicate that clicking it opens something else.